libraries/AP_Parachute/AP_Parachute.cpp

```diff
@@ -83,6 +83,14 @@ const AP_Param::GroupInfo AP_Parachute::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("OPTIONS", 7, AP_Parachute, _options, AP_PARACHUTE_OPTIONS_DEFAULT),
 
+    // @Param: TIMEOUT
+    // @DisplayName: Parachute timeout
+    // @Description: Triggers the parachute if the loss of control lasts for the time specified by this parameter
```

Comment:
Need to mention that it's copter only.

Reply:
maybe we should make the parameters only appear for copter..?

Reply:
@rmackay9 how to achieve it? I've tried with `APM_BUILD_COPTER_OR_HELI` but automatic checks failed.